dlib/image_saver/save_jpeg.cpp

```diff
@@ -10,14 +10,15 @@
 #include "../pixel.h"
 #include "save_jpeg.h"
 #include <stdio.h>
+#include <sstream>
+#include <setjmp.h>
+#include "image_saver.h"
+
 #ifdef DLIB_JPEG_STATIC
 #   include "../external/libjpeg/jpeglib.h"
 #else
 #   include <jpeglib.h>
 #endif
-#include <sstream>
```

Comment:
It is a include issue with windows, I have sent a PR which has been merged with the mainline dlib to correct it. https://github.com/davisking/dlib/pull/307